upi/openstack/bootstrap.yaml

```diff
@@ -38,5 +38,6 @@
   - name: 'Create the bootstrap floating IP'
     os_floating_ip:
       state: present
+      nat_destination: "{{ os_network }}"
```

Comment:
because the server might have multiple nics, "nat_destination" forces the floating ip to attach to the os_network